policyengine_core/data/dataset.py

```diff
@@ -451,43 +452,62 @@ def from_dataframe(dataframe: pd.DataFrame, time_period: str = None):
 
         return dataset
 
-    def upload_to_huggingface(self, owner_name: str, model_name: str):
+    def upload_to_huggingface(
+        self, owner_name: str, model_name: str, file_name: str
+    ):
         """Uploads the dataset to Hugging Face.
 
         Args:
             owner_name (str): The owner name.
             model_name (str): The model name.
         """
+
+        print(
+            f"Uploading to Hugging Face {owner_name}/{model_name}/{file_name}",
+            file=sys.stderr,
+        )
+
         token = os.environ.get(
```

Comment:
Could you also add the terminal prompt here, as I did in https://github.com/PolicyEngine/policyengine.py/pull/50? I think the duplication is OK

Reply:
Wrote a utility formula and some tests for it.